datastream-testcommon/src/main/java/com/linkedin/datastream/testutil/BaseKafkaZkTest.java

```diff
@@ -63,25 +69,42 @@ public void afterMethodTeardown() {
     _broker = null;
   }
 
-  protected static void createTopic(ZkUtils zkUtils, String topic, int partitionCount) {
-    if (!AdminUtils.topicExists(zkUtils, topic)) {
-      Properties properties = new Properties();
-      properties.put("message.timestamp.type", "LogAppendTime");
-      AdminUtils.createTopic(zkUtils, topic, partitionCount, 1, properties, null);
+  protected static void createTopic(AdminClient adminClient, String topic, int partitionCount) {
+    NewTopic newTopic = new NewTopic(topic, partitionCount, (short) 1);
+    try {
+      adminClient.createTopics(Collections.singletonList(newTopic)).all().get();
+    } catch (InterruptedException | ExecutionException e) {
+      if (e instanceof TopicExistsException || e.getCause() instanceof TopicExistsException) {
```

Comment:
do we know here for sure that InterruptedException or ExecutionException is just a one-layer wrapper around the TopicExistsException? Or would we need to recursively check the causes of the caught exception to know if it is TopicExistsException?